editor/modes/visual-editor/index.js

```diff
@@ -40,6 +52,20 @@ class VisualEditor extends Component {
 		}
 	}
 
+	onKeyDown( event ) {
+		const { uids } = this.props;
+		const isEditable = [ 'textarea', 'input', 'select' ].indexOf( document.activeElement.tagName.toLowerCase() ) !== -1
```

Comment:
I'm still a bit hesitant about this logic. Do you think this should be in a utility class, e.g. `isActiveElementEditable`, or `isEditableElement( document.activeElement )` ?

I'd found that in some browsers, `window.getSelection()` has a `type` property against which we could perhaps test `=== 'caret'` for reasonable accuracy, but unfortunately not all browsers define this property:

https://developer.mozilla.org/en-US/docs/Web/API/Selection/type